code/script.js

```diff
@@ -1,53 +1,230 @@
-// DOM selectors (variables that point to selected DOM elements) goes here 👇
-const chat = document.getElementById('chat')
+const chat = document.getElementById("chat")
+const inputWrapper = document.getElementById("input-wrapper")
+let choice = ""
+const audio = document.getElementById("botAudio")
+const audioUser = document.getElementById("userAudio")
 
-// Functions goes here 👇
+// Moved the greeting function here and added the form which we removed from HTML👇🏻
+const greetUser = () => {
+  showMessage(
+    `Hi, I'm Sharky 😊
+    Let's get started!
+    What's your name?`,
+    `bot`
+  )
+
+  inputWrapper.innerHTML = `<form id="name-form">
+    <input
+      placeholder="--------------------------------------------------------------"
+      id="name-input"
+      type="text"
+    />
+    <button id="send-btn" class="send-btn" type="submit">→</button>
+  </form>`
+  const sendBtn = document.getElementById("send-btn")
+  if (sendBtn) {
+    sendBtn.onclick = handleNameInput
+  }
+}
+// Moved the message function here 👇🏻
 
-// A function that will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
-  // The if statement checks if the sender is the user and if that's the case it inserts
-  // an HTML section inside the chat with the posted message from the user
-  if (sender === 'user') {
+  if (sender === "user") {
+    console.log("message is:", message)
+    console.log("sender is:", sender)
```

Comment:
Remove console.logs, they are used for development and shouldn't be in production versions.